src/components/Card/sass/Card.scss

```diff
@@ -0,0 +1,118 @@
+@use 'sass:map';
+.tyk-card { 
+  border: 1px solid theme-color('background', 'base');
+  border-radius: 8px;
+  box-shadow: 2px 2px 3px theme-color('background', 'base');
+  display: flex;
+  flex-direction: column;
+  overflow: hidden;
+
+  &__header {
+    align-items: center;
+    display: flex;
+    font-family: $font-inter-regular;
+    font-size: $sm-font-size;
+    padding: map.get($spacing, 'base') map.get($spacing, 'base') 0 map.get($spacing, 'base');
+  
+    &-right {
+      margin-left: auto;
+    }
+  }
+
+  &__title-group {
+    display: grid;
+    grid-template-columns: auto 1fr;
+    align-items: center;
+
+    .tyk-card__title-icon {
+      align-items: center; 
+      display: flex;
+      height: 42px;
+      justify-content: center;
+      margin-right: 10px;
+      grid-row: span 2;
+      width: 42px;
+    }
+  }
+
+  &__title { 
+    font-family: $font-inter-bold;
+    font-size: $lg-font-size;
+    display: box;
+    box-orient: vertical;
+    overflow: hidden;
+    margin: 0;
+    text-overflow: ellipsis;
+
+    &--max-1-lines {
+      line-clamp: 1;
+    }
+    &--max-2-lines {
+      line-clamp: 2;
+    }
+  }
+
+  &__sub-title { 
+    font-family: $font-inter-medium;
+    font-size: $xs-font-size;
+    margin: 0;
+  }
+
+  &__title-group + &__summary,
+  &__sub-title + &__summary,
+  &__title + &__summary{
+    margin-top: map.get($spacing, 'sm');
+  }
+
+  &__summary {
+    font-family: $font-inter-regular;
+    font-size: $sm-font-size;
+    display: -webkit-box;
+    -webkit-box-orient: vertical;
```

Comment:
These seem to be non standard and won't work the same in all the browsers. They should probably be replaced with flex css properties.